src/ecdh.rs

```diff
@@ -239,6 +289,17 @@ mod tests {
         });
         assert_eq!(result, Err(Error::CallbackPanicked));
     }
+
+    #[test]
+    #[should_panic(expected = "This is UB")]
+    // This test is technically underined behavior. but it's interesting to see if the behavior changes.
```

Comment:
s/underined/undefined

Reply:
Is it really a good idea to have this test? I mean, it could fail on other C implementations even, no?

Reply:
Yeah you're right. It was mainly me curious wehn/where/if this test will ever get broken (even though it's UB it's an indicator of the current instability of unwrapping through FFI, there are some proposals to just make this abort or something similar.

Reply:
Heh, yeah, we should probably remove this. Or maybe comment it out or something.